parglare/builder.py

```diff
@@ -0,0 +1,356 @@
+# -*- coding: utf-8 -*-
+import os
+from copy import deepcopy
+from typing import Dict, Union, List, Iterator, TypeVar, ContextManager, Optional, Callable, Any, Sequence
+
+from parglare.exceptions import GrammarBuilderValidationError
+from parglare.grammar import MULT_ONE_OR_MORE, MULT_ZERO_OR_MORE, MULT_OPTIONAL, Terminal, MULT_ONE
```

Comment:
parglare is using PEP-8. That means 79 line length. I'm ok to go up to 90.  You can [add flake8 line length setting in config.cfg like this](https://github.com/textX/textX/blob/master/setup.cfg#L81) and check your source for violations.